sources/web/datalab/idleTimeout.ts

```diff
@@ -95,9 +104,13 @@ function parseInterval(str: string) {
   str = str.replace(/\s*/, '');
   if (str) {
     logging.getLogger().debug('garbage left over in interval string: ' + str);
-    return NaN;
+    return { seconds: NaN, errorMessage: 'Invalid format' };
   }
-  return total;
+  if (total > 0 && total < idleTimeoutMinSeconds) {
+    const message = total + ' is smaller than minimum of ' + idleTimeoutMinSeconds;
```

Comment:
Units?  Nothing in the UI seems to say "seconds", and it is probably non-obvious given that the default is "90m"